addons/openebs/template/base/install.sh

```diff
@@ -374,6 +374,16 @@ function openebs_prompt_migrate_from_longhorn() {
     logWarn "    Detected Longhorn is running in the cluster. Data migration will be initiated to move data from Longhorn to storage class $OPENEBS_LOCALPV_STORAGE_CLASS."
     logWarn "    As part of this, all pods mounting PVCs will be stopped, taking down the application."
     logWarn "    It is recommended to take a snapshot or otherwise back up your data before proceeding."
+
+    semverParse "$KUBERNETES_VERSION"
```

Comment:
Should not the check be in the preflights (https://github.com/replicatedhq/kURL/blob/main/scripts/common/preflights.sh) ?

Reply:
I don't have strong opinion on the matter. I read the `preflight.sh` and it seems to contain a lot of checks regarding the node where we are planning to install, in the other hand this file here is pretty much "openebs" specific. If we move this from here to there we gonna have to migrate other logic that actually exist in other migrations. Yeah, I think it is better to keep it here but I am open to be convinced otherwise.

Reply:
So, that is fine. I do not have strong opinion either.
I do not think that is a blocker at all.